app/domain/errors.rb

```diff
@@ -514,6 +514,34 @@ module AuthnJwt
         msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
         code: "CONJ00108E"
       )
+
+      MappedClaimsMissingInput = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claims value is empty, or was not found.",
+        code: "CONJ00109E"
+      )
+
+      MappedClaimsBlankOrEmpty = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, one ore more mapping statements is blank or empty " \
+             "'{0-mapped-claims-value}'.",
+        code: "CONJ00110E"
+      )
+
+      MappedClaimInvalidFormat = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claim value '{0-mapped-claim-value}' is in invalid format."\
```

Comment:
@hilagross @shulifink 
Please review

Reply:
Failed to parse mapped claims, **the** value '{0-mapped-claim-value}' is in invalid format.